### ISSUE TITLE
Lauch RADx Mini Best Practices for Covid Test Kits

### ISSUE THREAD

**Issue by bruce-usab** [atbcb/usab-uswds#2331]
To do:
[ ] Pick a template
[ ] Scrub for hierarchy
[ ] Resolve any formatting inconsistencies 
[ ] Doublecheck and correct copy
[ ] add good alt text
[ ] USAB banner
[ ] USAB press releases

**Comment by bruce-usab** [comment]
Ping to @bryceeriksen -- please use this thread to communicate about action items.

**Comment by bruce-usab** [comment]
USAB used GitHub issues during migration, but that is not our habit nowadays.

@philbratta has the last two times, banner on USAB home page and press released via GovDelivery and SM.

**Comment by bruce-usab** [comment]
Awesome, @bryceeriksen I see you have a build churning away.  You have the pen!

I think the H6 can all be promoted to H5.  That can wait.

**Comment by bruce-usab** [comment]
@bryceeriksen  -- One limitation of our template is that H3 -- and only H3 -- are used to populate the left sidebar navigation menu.

Click/selecting the H3 in the menu will show the H4 under the parent h4.

Someday I hope to address this limitation, but I have been saying that for a couple of years now...

**Comment by bryceeriksen** [comment]
@bruce-usab ok! Good to know

**Comment by bryceeriksen** [comment]
@bruce-usab is it better to give images sizing (like px) or just full, large, medium etc?

**Comment by bruce-usab** [comment]
Ping to @dlynne -- to let me know when you are in.

**Comment by Dlynne** [comment]
I'm in!

**Comment by bruce-usab** [comment]
USWDS documentation is here;  https://designsystem.digital.gov/

**Comment by bruce-usab** [comment]
@dlynne -- i see you in us-access-board-gov Members.

**Comment by bruce-usab** [comment]
I just worked permissions for D'lynne.  I will see if there is any work for me starting about ~9am Eastern 12/13.  Left navigation sidebar is now repaired.

**Comment by bruce-usab** [comment]
I added some notes (links) to OP.

**Comment by bryceeriksen** [comment]
@bruce-usab I just sent an email with adjusted images.

**Comment by bruce-usab** [comment]
I will upload the replacement images (but not real soon).

@Dlynne -- I am not seeing your name on any of the recent builds:

Once signed into pages.cloud.gov, can you see this page: 
https://pages.cloud.gov/sites/852/builds

**Comment by Dlynne** [comment]
Yes I see what you mean. I can see the page ... but don't see my name
anywhere.
But I do see my changes reflected here:
https://github.com/Dlynne/usab-uswds/blob/radX/_tad/rad-x.md

Is this not the right place?

D'

**Comment by bruce-usab** [comment]
I am ready to pull to Beta which will give everyone a cleaner URL to review.  Ping to @Dlynne and @bryceeriksen as editing URLs will also change...

**Comment by bruce-usab** [comment]
Per email, page is now in staging/beta.  I will make document live once the alt tags are done.
Ping to @Dlynne @bryceeriksen @bstinnette 

- File in repo is:  https://github.com/atbcb/usab-uswds/blob/beta/_tad/rad-x.md
- Preview on staging is:  https://beta.access-board.gov/tad/radx/

I will delete the radX brach we have been working on.

**Comment by bruce-usab** [comment]
@Dlynne asked if adding inline CSS is okay:  yes, whatever you need to do.

Most of the classes are inherited from USWDS.  The custom CSS we are using is in this file:
https://github.com/atbcb/usab-uswds/blob/beta/_assets/css/styles.scss

If you want to define and reference some custom classes, that is fine too.

The USWDS stuff is pretty good at responsive design, but it is not perfect.  If you feel like some of the page is breaking weird on larger or smaller screens, please see how other pages on the side do at those screen sizes.  There might not be easy fixes.

**Comment by bruce-usab** [comment]
Front matter is brittle.   @Dlynne and @bryceeriksen , you have the pen.

If you want to replace an image, remember to delete it first.  Otherwise, GitHub will go through the whole upload churn and _really_ act like it has replaced the image.

**Comment by bruce-usab** [comment]
I have added the logo to the right sidebar.  If you want to play with that, the file is:
- https://github.com/atbcb/usab-uswds/blob/beta/_includes/sidenav-right-special.html

**Comment by Dlynne** [comment]
Thank you, Bruce!!

**Comment by bryceeriksen** [comment]
@bruce-usab I added this markup to the to adjust the margin issue css: 
.tablet\:grid-col p {
    margin: 5px;
}

Is this an expectable solution or will it mess with things elsewhere?

**Comment by bruce-usab** [comment]
@bryceeriksen  that looks good to me.

@Dlynne i have deleted the "special" right side bar for a more purpose-built instance.
https://github.com/atbcb/usab-uswds/blob/beta/_includes/sidenav-right-radx.html
